tidb-lightning/tidb-lightning-compatibility-and-scenarios.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Compatibility of TiDB Lightning and IMPORT INTO with TiCDC and Log Backup
+summary: Learn about compatibility of IMPORT INTO and TiDB Lightning with log backup and TiCDC.
+---
+
+# Compatibility of TiDB Lightning and IMPORT INTO with TiCDC and Log Backup
+
+This document describes TiDB Lightning and `IMPORT INTO` compatibility with [log backup](/br/br-pitr-guide.md), [TiCDC](/ticdc/ticdc-overview.md), as well as some special usage scenarios.
+
+## `IMPORT INTO` vs. TiDB Lightning
+
+[`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) currently integrates with the physical import mode of TiDB Lightning, but there are some differences. See [`IMPORT INTO` vs. TiDB Lightning](/tidb-lightning/import-into-vs-tidb-lightning.md) for details.
+
+## Compatibility with log backup and TiCDC
+
+- TiDB Lightning [logical import mode](/tidb-lightning/tidb-lightning-logical-import-mode.md) is compatible with log backup and TiCDC.
+
+- TiDB Lightning [physical import mode](/tidb-lightning/tidb-lightning-physical-import-mode.md) is not compatible with log backup and TiCDC. The reason is that physical import mode directly ingests the encoded KV Pairs of the source data to TiKV, and TiKV will not generate the corresponding change log during this process. Without this part of the change log, the relevant data cannot be backed up and cannot be replicated by TiCDC.
+
+- `IMPORT INTO` is not compatible with log backup and TiCDC.
+
+## Scenarios for TiDB Lightning physical import mode
+
+This section describes how to use TiDB Lightning together with [log backup](/br/br-pitr-guide.md) and [TiCDC](/ticdc/ticdc-overview.md).
+
+If the performance of TiDB Lightning logical import can meet the performance requirements of the application and the application requires TiDB Lightning imported tables to be backed up or replicated downstream using TiCDC, it is recommended to use TiDB Lightning logical import mode.
+
+If the application requires import performance and can only use TiDB Lightning physical import mode, and the tables need to be backed up or replicated downstream using TiCDC, the following scenarios are recommended.
+
+### Used with log backup
+
+#### Scenario 1: tables in physical import mode do not need to be backed up
+
+In this scenario, the compatibility check will report an error after starting TiDB Lightning because [PITR](/br/br-log-architecture.md#pitr) is enabled. If you are sure that these tables do not need to be backed up or backed up by [log backup](/br/br-pitr-guide.md), you can change the [TiDB Lightning configuration file](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-task) in the `Lightning.check-requirements` parameter to `false` and restart the task.
```

Comment:
```suggestion
In this scenario, the compatibility check will report an error after starting TiDB Lightning because [PITR](/br/br-log-architecture.md#process-of-pitr) is enabled. If you are sure that these tables do not need to be backed up or backed up by [log backup](/br/br-pitr-guide.md), you can change the [TiDB Lightning configuration file](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-task) in the `Lightning.check-requirements` parameter to `false` and restart the task.
```